pkg/bootstrap/platform/azure.go

```diff
@@ -187,10 +187,31 @@ func (e *azureEnv) azureName() string {
 // Returns the Azure tags
 func (e *azureEnv) azureTags() map[string]string {
 	tags := map[string]string{}
+	if tl, ok := e.computeMetadata["tagsList"]; ok {
+		tlByte, err := json.Marshal(tl)
```

Comment:
Is there any reason not to simply `json.Unmarshal(byte[](tl), &atl)`?

Reply:
tl is an `interface`. `Marshal` is the way to convert to bytes.

Reply:
I see thanks!